server/graphql/v1/types.js

```diff
@@ -1693,16 +1693,11 @@ export const OrderType = new GraphQLObjectType({
             description: 'type of transaction (DEBIT/CREDIT)',
           },
         },
-        resolve(order, args, req) {
-          const query = {
-            where: {},
-            limit: args.limit || 10,
-            offset: args.offset || 0,
-          };
-          if (args.type) {
-            query.where.type = args.type;
-          }
-          return req.loaders.Transaction.findByOrderId(query).load(order.id);
+        async resolve(order, args, req) {
```

Comment:
Aligned this with GQLV2. As mentioned, the previous data loader was not working. We're only using this on the confirm order page, so performance shouldn't be an issue.